### PR TITLE
PlatformService: AudioVideoIVRSample and moved all common files to Qu…

### PR DESCRIPTION
**Pull request by kuarora9091**
…ickSamplesCommon

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample.sln](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample.sln) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/.gitignore](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/.gitignore) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/Startup.Auth.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/Startup.Auth.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/WebApiConfig.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/WebApiConfig.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRSample.csproj](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRSample.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Controllers/CallbackController.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Controllers/CallbackController.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/prompt.wav](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/prompt.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptOne.wav](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptOne.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptTwo.wav](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptTwo.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Startup.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Startup.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Debug.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Debug.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Release.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Release.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/packages.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/packages.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/DiagnosticLogger.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/DiagnosticLogger.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/LoggingContextActionFilter.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/LoggingContextActionFilter.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/SimpleEventChannel.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/SimpleEventChannel.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/UnityHelper.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/UnityHelper.cs) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C21%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703212016574066-230%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("CallCenterSample")]
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs]: Update

File: Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("CallCenterSample")]
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs]: Update

File: Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.config [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+<!--
+  For more information on how to configure your ASP.NET application, please visit
+  http://go.microsoft.com/fwlink/?LinkId=301879
+  -->
+<configuration>
+  <configSections />
+  <connectionStrings />
+  <appSettings>
+    <!--Do not put a "/" at the end of uri-->
+    <add key="MyAudienceUri" value="https://audiovideoivrsample.metio.onmicrosoft.com" />
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.config]: Please do not put values for any of the keys here

File: Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj [hunk 4]
```diff
@@ -67,6 +67,10 @@
       <HintPath>..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
+	<Reference Include="Microsoft.Practices.Unity, Version=4.0.0.0, Culture=neutral, PublicKeyToken=6d32ff45e0ccc69f, processorArchitecture=MSIL">
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj]: Is this a new nuget dependency of QuickSamplesCommon project? If yes, please update packages.config. Use Nuget package manager to update packages.config

File: Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj [hunk 5]
```diff
@@ -85,6 +89,9 @@
       <HintPath>..\packages\System.Net.Http.Formatting.Extension.5.2.3.0\lib\System.Net.Http.Primitives.dll</HintPath>
       <Private>True</Private>
     </Reference>
+	    <Reference Include="System.Web.Http, Version=5.2.3.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (sankum-msft) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj]: Is this a new nuget dependency of QuickSamplesCommon project? If yes, please update packages.config.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample.sln](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample.sln) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/.gitignore](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/.gitignore) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/Startup.Auth.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/Startup.Auth.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/WebApiConfig.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/WebApiConfig.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRSample.csproj](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRSample.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Controllers/CallbackController.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Controllers/CallbackController.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/prompt.wav](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/prompt.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptOne.wav](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptOne.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptTwo.wav](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptTwo.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Startup.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Startup.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Debug.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Debug.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Release.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Release.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/packages.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/packages.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/DiagnosticLogger.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/DiagnosticLogger.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/LoggingContextActionFilter.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/LoggingContextActionFilter.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/SimpleEventChannel.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/SimpleEventChannel.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/UnityHelper.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/UnityHelper.cs) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C22%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703220122149003-230%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/.gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+AudioVideoIVRSample.csproj.user
```
Comment (kuarora9091) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/.gitignore]: removed.

File: Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+using System.Threading.Tasks;
+using Microsoft.Practices.Unity.Configuration.ConfigurationHelpers;
+using Microsoft.Rtc.Internal.RestAPI.ResourceModel;
+using Microsoft.Rtc.Internal.Platform.ResourceContract;
+
+namespace AudioVideoIVRSample
+{
+    public class AudioVideoIVRJob
+    {
+        private readonly Uri _callbackUri;
```
Comment (kuarora9091) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs]: done

File: Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+                }
+            }
+            );
+        }
+
+
+
+        private async Task StartAudioVideoIVRFlowAsync(IncomingInviteEventArgs<IAudioVideoInvitation> e)
+        {
+            Logger.Instance.Information(string.Format("[StartAudioVideoIVRFlowAsync] StartCallCenterFlowAsync: LoggingContext: {0}", _loggingContext));
```
Comment (kuarora9091) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs]: done

File: Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (kuarora9091) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs]: done

File: Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj (hunk 5 above)
Comment (kuarora9091) [inline Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj]: added <package id="Microsoft.AspNet.WebApi" version="5.2.3" targetFramework="net462" /> to packges

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample.sln](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample.sln) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/Startup.Auth.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/Startup.Auth.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/WebApiConfig.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/App_Start/WebApiConfig.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRJob.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRSample.csproj](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/AudioVideoIVRSample.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Controllers/CallbackController.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Controllers/CallbackController.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Global.asax.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Properties/AssemblyInfo.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/prompt.wav](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/prompt.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptOne.wav](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptOne.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptTwo.wav](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Resources/promptTwo.wav) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Startup.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Startup.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Debug.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Debug.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Release.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.Release.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/Web.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/packages.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample/packages.config) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/DiagnosticLogger.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/DiagnosticLogger.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/LoggingContextActionFilter.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/LoggingContextActionFilter.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/QuickSamplesCommon.csproj) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/SimpleEventChannel.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/SimpleEventChannel.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/UnityHelper.cs](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/UnityHelper.cs) | :white_check_mark:Succeeded | |
[Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/packages.config](https://github.com/OfficeDev/skype-docs/blob/kuarora/master/Skype/Trusted-Application-API/samples/QuickStartSamples/QuickSamplesCommon/packages.config) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C3%5C22%5C5315e434-bbfc-97f8-0e56-7036cb0b5e04%5CPullRequest%5C201703221809572247-230%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**